dcos/package.py

```diff
@@ -943,6 +944,25 @@ def copy_to_cache(self, target_dir):
                 'Unable to fetch packages from [{}]'.format(self.url))
 
 
+def _onerror(func, path, exc_info):
```

Comment:
This still doesn't follow the pydoc format used in other functions.  Please follow that format.